src/api/entities/CurrentAccount.ts

```diff
@@ -73,6 +74,15 @@ export class CurrentAccount extends Account {
       hasTokens = tokens.length === 0 || !differenceBy(tokens, currentTokens, 'ticker').length;
     }
 
+    let hasTransactionGroups: boolean;
+    if(currentTransactionGroups === null) {
+      hasTransactionGroups = true;
+    } else if (transactionGroups === null ) {
+      hasTransactionGroups = false;
+    } else {
+      hasTransactionGroups = transactionGroups.length === 0 || !difference(transactionGroups, currentTransactionGroups).length
```

Comment:
@monitz87 I implemented it this so it's consistent with the rest, but the value returned by `this.getPermissions` is `[]` which implies no permission granted for primary key, is this right?

https://github.com/PolymathNetwork/polymesh-sdk/blob/a0872cf42ccbf170d779a6d14bcb69b943311e82/src/api/entities/CurrentAccount.ts#L41